src/commands/force/org/display.ts

```diff
@@ -90,14 +90,20 @@ export class OrgDisplayCommand extends SfdxCommand {
     const result = (
       await OrgListUtil.retrieveScratchOrgInfoFromDevHub(hubOrg.getUsername(), [sfdc.trimTo15(orgId)])
     )[0];
-    return {
-      status: result.Status,
-      expirationDate: result.ExpirationDate,
-      createdBy: result.CreatedBy?.Username,
-      edition: result.Edition ?? undefined, // null for snapshot orgs, possibly others.  Marking it undefined keeps it out of json output
-      namespace: result.Namespace ?? undefined, // may be null on server
-      orgName: result.OrgName,
-      createdDate: result.CreatedDate,
-    };
+
+    if (result) {
+      return {
+        status: result.Status,
+        expirationDate: result.ExpirationDate,
+        createdBy: result.CreatedBy?.Username,
+        edition: result.Edition ?? undefined, // null for snapshot orgs, possibly others.  Marking it undefined keeps it out of json output
+        namespace: result.Namespace ?? undefined, // may be null on server
+        orgName: result.OrgName,
+        createdDate: result.CreatedDate,
+      };
+    }
+    throw new SfdxError(
+      `No information for scratch org with ID ${sfdc.trimTo15(orgId)} found in Dev Hub ${hubOrg.getUsername()}`
```

Comment:
is there anything the user can do to get past this error the next time they run the command? I think it would be nice to add an action so they don't get this error the next time they run the command - if possible

Reply:
Good call.  I **think** you could delete the org if you're no longer using it but the hub lost it somehow?

I'm not exactly sure how the user got this error so I'm not sure what the fix might be.  We could definitely tell them to check the ScratchOrgInfo to see if the org is there?